cranelift/filetests/filetests/runtests/simd-umulhi.clif

```diff
@@ -0,0 +1,29 @@
+test interpret
```

Comment:
Since this isn't testing the SIMD version of these instructions, can we rename this to `umulhi.clif`?